src/utils/uint256.ts

```diff
@@ -26,6 +46,20 @@ export function isUint256(bn: BigNumberish): boolean {
 /**
  * Convert BigNumberish (string | number | bigint) to Uint256 (hex)
  * Legacy support Export
+ * @param bn BigNumberish value to convert to Uint256 (hex)
+ * @returns {Uint256} Uint256 (hex) representation of the input BigNumberish value
+ * @example
+ * ```typescript
+ * import { uint256 } from "path/to/file";
```

Comment:
This time we have import of the namespace. Here it's correct...
except that its `from 'starknet'`.